README.md

```diff
@@ -49,6 +49,8 @@ The following packages are required to build Telegrand:
 - TDLib 1.8.0
 - [Telegram API Credentials](https://my.telegram.org/) (optional, but recommended)
 
+Also, Telegrand requires the Libav GStreamer plugin installed in your system to correctly show media files.
```

Comment:
We really need other people to check whether that is enough. Here on Fedora 35, at least, it seems I need `gstreamer1-plugins-base` `gstreamer1-plugins-good`, `gstreamer1-plugins-bad-free` in addition to `gstreamer1-libav` from rpmfusion. If one of them is missing the animation is not shown.

Reply:
Mh, weird, at least by looking at gst docs it seems that GIF and MPEG4 decoding are handled by the libav plugin. I'll investigate a bit more.

Reply:
I finally tested on arch and it seems that they are all required indeed.